### PR TITLE
[DEBUG] Add some verbose logging

### PR DESCRIPTION
**Pull request by lackstein**

### CODE REVIEW

File: library/Hasql/Pool.hs [hunk 1]
```diff
@@ -105,52 +117,63 @@ use Pool {..} sess = do
       return $ return False
   join . atomically $ do
     reuseVar <- readTVar poolReuse
+    connectionPoolEmpty <- isEmptyTQueue poolConnectionQueue
+    cap <- readTVar poolCapacity
+    let logState :: String -> IO ()
+        logState msg = logger $ msg <> " - Connection pool empty: " <> show connectionPoolEmpty <> " - Available pool capcity: " <> show cap
```
Comment (elldritch) [inline library/Hasql/Pool.hs]:
```suggestion
        logState msg = logger $ msg <> " - Connection pool empty: " <> show connectionPoolEmpty <> " - Available pool capacity: " <> show cap
```

**Review by elldritch — APPROVED** [review]
Logging functionality LGTM, but like with the Faktory worker, I'd prefer fewer style diffs.

**Comment by lackstein** [comment]
@elldritch I'm not planning to merge this in. If we think it's helpful past debugging this issue, we should modify the library to use our logging system.

**Comment by lackstein** [comment]
Closing this since we're no longer debugging this issue.

**Pull request closed without merging**